src/python/CRABInterface/HTCondorDataWorkflow.py

```diff
@@ -170,51 +182,167 @@ def _compactLumis(datasetInfo):
             return lumilist
 
         res = {}
-        self.logger.info("About to compute report of workflow: %s with usedbs=%s. Getting status first." % (workflow, usedbs))
-        statusRes = self.status(workflow, userdn)[0]
 
-        #get the information we need from the taskdb/initilize variables
+        ## Get the information we need from the Task DB.
         row = next(self.api.query(None, None, self.Task.ID_sql, taskname = workflow))
         row = self.Task.ID_tuple(*row)
+        userWebDirURL = row.user_webdir
```

Comment:
The userwebdir can be `None` since it is updated in `AdjustSites` and in case cmsweb is not answering we do not consider it a permanet error that halts the task: https://github.com/dmwm/CRABServer/blob/master/scripts/AdjustSites.py#L309-L315

We should probably make this type of error prmanent, but for the moment we should not assume it exists.